fuzz-tests/src/bin/decimal.rs

```diff
@@ -0,0 +1,40 @@
+#![cfg_attr(feature = "libfuzzer-sys", no_main)]
+
+use arbitrary::Arbitrary;
+#[cfg(feature = "libfuzzer-sys")]
+use libfuzzer_sys::fuzz_target;
+
+use radix_engine_common::math::*;
+
+#[derive(Debug, Arbitrary)]
+struct OneDecimal(Decimal, Decimal, i64, u32, i32, RoundingMode);
+
+// Fuzzer entry points
+#[cfg(feature = "libfuzzer-sys")]
+fuzz_target!(|decimal: OneDecimal| {
+    let _ = decimal.0.checked_sqrt();
+    let _ = decimal.0.is_positive();
+    let _ = decimal.0.is_negative();
+    let _ = decimal.0.is_zero();
+    let _ = decimal.0.checked_abs();
+    let _ = decimal.0.checked_cbrt();
+    let _ = decimal.0.checked_ceiling();
+    let _ = decimal.0.checked_floor();
+    let _ = decimal.0.checked_neg();
+    if decimal.4 >= 0 && decimal.4 <= Decimal::SCALE as i32 {
+        let _ = decimal.0.checked_round(decimal.4, decimal.5);
+    }
+    let _ = decimal.0.checked_add(decimal.1);
+    let _ = decimal.0.checked_sub(decimal.1);
```

Comment:
would also try 
```
decimal.1.checked_sub(decimal.0)
decimal.1.checked_div(decimal.0)
```

Reply:
Hrm, not sure if testing this ourselves is useful since it's the fuzzer's job to find interesting cases?

Reply:
true. makes sense